copyright

```diff
@@ -605,12 +695,35 @@ Files:
  images/land/station1*
  images/land/station2*
  images/land/station3*
- images/land/station4*
```

Comment:
Looks like station4 was removed from the NASA copyright but wasn't readded anywhere else.